src/languages.js

```diff
@@ -11,6 +11,23 @@ const supportedLanguages = {
 };
 export const savedLanguage = localStorage.getItem("preferredLanguage") || "en";
 const youtubeOpts = "?rel=0&autoplay=1";
+const whatsappMapsUrl =
+	"https://uclexcites.blog/2024/06/26/whatsapp-maps-connecting-users-and-producers-of-ground-information/";
+const extremeCitizenUrl = "https://www.youtube.com/watch?v=IgQc7GQ1m_Y";
+const marcosUrl =
```

Comment:
I feel like there should be a better, structured way to organise contributor details so we can iterate over a list of objects and display name, role, link etc. Maybe one for another issue.

Reply:
have we opened another issue or not yet?

Reply:
Done #100